pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+<!-- Please add the WIP label if the PR isn't complete. -->
```

Comment:
People can now create draft PRs so we should probably go with that rather than WIP label. (those have UI that prevents merging)

Reply:
👍